src/coreclr/vm/exceptionhandling.h

```diff
@@ -673,10 +563,6 @@ private: ;
 
     void ReleaseResources();
```

Comment:
This should not be needed.

Reply:
It's a bit harder to reason about this since it's not purely unreferenced code.

Reply:
Basically anything that references non-static methods in the ExceptionTracker is a dead code. No ExceptionTracker instance gets ever created with the new EH. The ExInfo has replaced it.

Reply:
Makes sense.